<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>691</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1034</width>
    <height>691</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SimulPad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/logoIcon.png</normaloff>:/resources/logoIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignHCenter">
     <widget class="QWidget" name="docFrame" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>750</width>
        <height>550</height>
       </size>
      </property>
      <layout class="QGridLayout" name="docFrameLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="MyTextEdit" name="textEditShared">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>750</width>
           <height>550</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
           &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
           p, li { white-space: pre-wrap; }
           &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Calibri'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
           &lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="verticalLine">
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="sideLayout">
      <item>
       <widget class="QFrame" name="myTag">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>258</width>
          <height>92</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>258</width>
          <height>92</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="myUsername">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>10</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Gill Sans MT</family>
           <pointsize>17</pointsize>
          </font>
         </property>
         <property name="text">
          <string>dario.patti90</string>
         </property>
        </widget>
        <widget class="QLabel" name="myStatus">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>60</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Online</string>
         </property>
        </widget>
        <widget class="QLabel" name="myAvatar">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>71</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/resources/avatar.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="myLed">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>60</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/resources/greenLed.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="onlineRollButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>258</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>258</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">text-align:left;</string>
        </property>
        <property name="text">
         <string>Online Users</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/resources/arrow_up.png</normaloff>:/resources/arrow_up.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listOnlineUsers">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>258</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>258</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="offlineRollButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>258</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>258</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">text-align:left;</string>
        </property>
        <property name="text">
         <string>Offline Users</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/resources/arrow_up.png</normaloff>:/resources/arrow_up.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listOfflineUsers">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>258</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>258</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>SynkEditor</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderlined"/>
   <addaction name="separator"/>
   <addaction name="actionAlignLeft"/>
   <addaction name="actionAlignCenter"/>
   <addaction name="actionAlignRight"/>
   <addaction name="actionAlignJustify"/>
   <addaction name="actionTestColor"/>
   <addaction name="actionTestCursor"/>
   <addaction name="actionTestTag"/>
   <addaction name="actionTestDisconnect"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionExport_to_PDF"/>
    <addaction name="actionInvite"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/ic_content_copy_36pt_2x.png</normaloff>:/resources/ic_content_copy_36pt_2x.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/ic_content_cut_36pt_2x.png</normaloff>:/resources/ic_content_cut_36pt_2x.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/ic_content_paste_36pt_2x.png</normaloff>:/resources/ic_content_paste_36pt_2x.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/ic_format_bold_36pt_2x.png</normaloff>:/resources/ic_format_bold_36pt_2x.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/ic_format_italic_36pt_2x.png</normaloff>:/resources/ic_format_italic_36pt_2x.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderlined">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/ic_format_underlined_36pt_2x.png</normaloff>:/resources/ic_format_underlined_36pt_2x.png</iconset>
   </property>
   <property name="text">
    <string>Underlined</string>
   </property>
  </action>
  <action name="actionAlignLeft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/ic_format_align_left_36pt_2x.png</normaloff>:/resources/ic_format_align_left_36pt_2x.png</iconset>
   </property>
   <property name="text">
    <string>AlignLeft</string>
   </property>
  </action>
  <action name="actionAlignCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/ic_format_align_center_36pt_2x.png</normaloff>:/resources/ic_format_align_center_36pt_2x.png</iconset>
   </property>
   <property name="text">
    <string>AlignCenter</string>
   </property>
  </action>
  <action name="actionAlignRight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/ic_format_align_right_36pt_2x.png</normaloff>:/resources/ic_format_align_right_36pt_2x.png</iconset>
   </property>
   <property name="text">
    <string>AlignRight</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect...</string>
   </property>
  </action>
  <action name="actionExport_to_PDF">
   <property name="text">
    <string>Export to PDF</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAlignJustify">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/ic_format_align_justify_36pt_2x.png</normaloff>:/resources/ic_format_align_justify_36pt_2x.png</iconset>
   </property>
   <property name="text">
    <string>AlignJustify</string>
   </property>
  </action>
  <action name="actionInvite">
   <property name="text">
    <string>Invite to Collaborate</string>
   </property>
  </action>
  <action name="actionTestCursor">
   <property name="text">
    <string>TestCursor</string>
   </property>
  </action>
  <action name="actionTestTag">
   <property name="text">
    <string>TestTag</string>
   </property>
  </action>
  <action name="actionTestDisconnect">
   <property name="text">
    <string>TestDisconnect</string>
   </property>
   <property name="toolTip">
    <string>TestDisconnect</string>
   </property>
  </action>
  <action name="actionTestColor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/colorButtonIcon.png</normaloff>:/resources/colorButtonIcon.png</iconset>
   </property>
   <property name="text">
    <string>TestColor</string>
   </property>
   <property name="toolTip">
    <string>Highlights the text based on who entered it</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyTextEdit</class>
   <extends>QTextEdit</extends>
   <header>mytextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
